crates/autopilot/src/driver_model.rs

```diff
@@ -112,19 +112,34 @@ pub mod solve {
         pub signature: Signature,
     }
 
-    #[derive(Clone, Debug, Serialize)]
+    #[derive(Clone, Debug, Serialize, Deserialize)]
     #[serde(rename_all = "lowercase")]
     pub enum Class {
         Market,
         Limit,
         Liquidity,
     }
 
+    #[derive(Clone, Debug, Default, Deserialize)]
+    #[serde(deny_unknown_fields)]
+    pub struct Reward {
+        /// The address to which the CIP20 performance reward will be sent for
+        /// this solution. Usually the solver address used for
+        /// submitting the solution onchain.
+        pub performance_address: H160,
+        /// The address to which the CIP20 participation reward will be sent for
+        /// this solution. Usually the solver address used for
+        /// submitting the solution onchain.
+        pub participation_address: H160,
+    }
```

Comment:
Do we actually need this info? Don't solvers get payed out to whatever address is configured at the time of payout and not at the time of solution submission?

Reply:
This was introduced because we need to save the participants for each auction in the `auction_participants` table.
Seemed easier and more general to do it this way (solvers can dynamically decide and change the payout addresses without our action). 
But we could definitely do it differently, like having a map <solver, address> locally in autopilot if we want to remove `Reward` from the driver interface...

Reply:
I substituted `Reward` with `SubmissionAddress` to simplify things. (a lot of comments came from the confusion I introduced with this new field).